frontend/src/components/notFound/not-found.css

```diff
@@ -0,0 +1,16 @@
+.container_not_found {
```

Comment:
I understand your point for this, but this is out of scope for now. Let's remove it too.

Reply:
As well as the asset you uploaded (not found image).